From cc505d90bbff05087984e4381cf483307077b269 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 25 Sep 2017 22:29:19 +0800
Subject: [PATCH 2566/5242] MLK-16375-3 video: fbdev: dcss: add
 '__maybe_unused' to 'fill_db'
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

commit  22680b3e9ff52127354b99f8aac44dfeb3d87464 from
https://source.codeaurora.org/external/imx/linux-imx.git

The function 'fill_db()' is not used at this moment
which caused gcc compiler generate the build warning:

"‘fill_db’ defined but not used [-Wunused-function]"

So add attribute '__maybe_unused' to 'fill_db()' functon
definition to avoid this build warning.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/imx_dcss.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/video/fbdev/mxc/imx_dcss.c b/drivers/video/fbdev/mxc/imx_dcss.c
index 790be11..35aaf36 100644
--- a/drivers/video/fbdev/mxc/imx_dcss.c
+++ b/drivers/video/fbdev/mxc/imx_dcss.c
@@ -674,9 +674,9 @@ static void fill_sb(struct cbuffer *cb,
 	cb->sb_data_len++;
 }
 
-static void fill_db(struct cbuffer *cb,
-		    uint32_t offset,
-		    uint32_t value)
+static void __maybe_unused fill_db(struct cbuffer *cb,
+				   uint32_t offset,
+				   uint32_t value)
 {
 	struct ctxld_unit *unit = NULL;
 
-- 
1.7.9.5

